features/storage/kvstore/tdbstore/TDBStore.cpp

```diff
@@ -24,7 +24,10 @@
 #include "mbed_error.h"
 #include "mbed_wait_api.h"
 #include "MbedCRC.h"
+//Bypass the check of NVStore co existance if compiled for TARGET_TFM
+#if !(TARGET_BYPASS_NVSTORE_CHECK)
```

Comment:
remove `TARGET_` prefix as it is not applicable.
```suggestion
#if !(BYPASS_NVSTORE_CHECK)
```

Reply:
Yes, I had something similar on my mind. Similar what we do in storage mbed_lib.json files (NVSTORE_MAX_KEYS or similar macros). This could be TBDSTORE_BYPASS_NVSTORE_CHECK (prefix TBDSTORE as it is only applicable in this storage componenet) or as alex suggests?

Reply:
I have accepted @alzix suggestion.